hq/app/views/iam/iamCredReportBody.scala.html

```diff
@@ -2,10 +2,17 @@
 @import model._
 @import logic.CredentialsReportDisplay
 @(report: CredentialReportDisplay)
-
+@message = @{s"Report generated at ${DateUtils.printTime(report.reportDate)} (refreshed every four hours)"}
```

Comment:
@philmcmahon what about declaring constants like this in templates?

Reply:
I think this is a neat solution. The alternative would be to store the strings in the scala codebase and pass through to the controller but since they're just strings related to the view that the app doesn't care about I think it's reasonable to put them in the view